test/server/tests.js

```diff
@@ -1,5 +1,15 @@
 import { agent } from "supertest";
+import { graphql } from "graphql";
 import { run, stop } from "../../server/server";
+import { Schema, root } from "../../server/gql/schema";
+import { queries } from "./queries";
+
+const assert = condition => {
```

Comment:
This could take a second argument with a message to return in the error if the condition is not met